spirv_glsl.cpp

```diff
@@ -804,6 +804,9 @@ bool CompilerGLSL::ssbo_is_std430_packing(const SPIRType &type)
 
 string CompilerGLSL::layout_for_variable(const SPIRVariable &var)
 {
+	if (is_legacy_es() || (!options.es && options.version < 330))
```

Comment:
Indeed... I was reading up on the 420 pack in the meantime and realize now that this is too general.

Problem is on OS X there's no 420 pack (and probably never will be) - so anything below 330 won't like layout qualifiers and this check is actually just fine. 

How about a compiler option to enable/disable the use of the 420 pack?